### PR TITLE
Feature/header menu

### PR DESCRIPTION
**Pull request by GerardoGomezG**
### Personal Portfolio: Mobile version

imagen

Features:
- Header with logo
- Menu button
- Developer's name
- Brief information about the developer

### CODE REVIEW

**Comment by danyhoshi** [comment]
# Great! 😄 

I hope the hamburger icon transition in the future. Is a good idea your way! Can you share the reference for it?
Only suggest adding an avatar or some image profile like the original design.

File: styles.css [hunk 1]
```diff
@@ -1,4 +1,92 @@
...
+  transform: translate(0, 11px) rotate(-45deg);
+}
+
+/* Fade out the second bar with JavaScript - JS code pending */
+.change .menu-button-bar2 { opacity: 0; }
+
+/* Rotate last bar with JavaScript - JS code pending */
+.change .menu-button-bar3 {
+  transform: translate(0, -11px) rotate(45deg);
+}
```
Comment (danyhoshi) [inline styles.css:45]: Hi! @GerardoGomezG Great job! This section is a good idea, Can you share where you got it from? 😄

**Review by danyhoshi — APPROVED** [review]
Great! only I have one question I leave about the hamburger icon 😄 You can merge!

File: styles.css (hunk 1 above)
Comment (GerardoGomezG) [reply styles.css:45]: Hello @danyhoshi !

You'll find the requested info here: [Hamburger button](https://blog.hubspot.com/website/hamburger-button)

Thanks

**Pull request merged**